requirements.txt

```diff
@@ -1,3 +1,4 @@
+anyio
```

Comment:
I don't think we need to do this. anyio is a dependency of starlette, which is installed by fastapi.

Reply:
OK, thanks.